.vscode/c_cpp_properties.json

```diff
@@ -4,7 +4,9 @@
             "name": "Win32",
             "includePath": [
                 "${workspaceFolder}/build",
-                "${workspaceFolder}/**"
+                "${workspaceFolder}/**",
+                "${workspaceFolder}/vendor/quill/include",
+                "${workspaceFolder}/vendor/quill/include"
```

Comment:
why is this doubled?

Reply:
It shouldn't be commited in the first place